src/drive/web/modules/views/Recent/index.spec.jsx

```diff
@@ -96,5 +100,11 @@ describe('Recent View', () => {
     expect(
       parentDiv1.getElementsByClassName('fil-file-path')[0].textContent
     ).toEqual(path)
+
+    // navigates  to the history view
+    const historyItem = getByText('History')
+    fireEvent.click(historyItem)
+
+    expect(findByText('Current version')).resolves.toBeTruthy()
```

Comment:
This expect must be awaited

await expect(findByText('Current version')).resolves.toBeTruthy()